otwcode/otwarchive
# AO3-6410 Update the automated PR labeller to handle the .github directory better.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6410

## Purpose

- Make sure to include files that start with a `.` in the labeller checks for the Awaiting Review and Coder Has Actioned Review labels.
- Switch to a different syntax for the Tests Only label to try to improve handling of the `.github` directory.

## Testing Instructions

Once this PR has been merged, set one of the dependabot PRs to "Reviewed: Action Needed," and then tell dependabot to rebase the PR. If it adds the "Coder Has Actioned Review" label and the "Scope: Tests Only" label, that should indicate that the two checks will work on future PRs.

Note that because the PR labeller is set to run as `pull_request_target`, the changes in this PR will not be visible on this PR, only on PRs labelled after this PR is merged.
